cmd/cl-adm/cmd/create/create_peer.go

```diff
@@ -37,15 +37,20 @@ type PeerOptions struct {
 	DataplaneType string
 	// LogLevel is the log level.
 	LogLevel string
+	// RemoteImage determines whether to use a local image or pull it from a container registry.
+	RemoteImage bool
```

Comment:
```suggestion
```

Let's remove this argument.
An empty `ContainerRegistry` will denote a local image. All other values denote a remote image.

Reply:
I think we need to keep him because most of the users will use the remote registry and it is much easier to add the flag  --remote-image instead of  container-registry=ghcr.io/clusterlink-net/

Reply:
Are we planning to release our images to `gchr.io`?
In that case, let's set the default registry to gchr, and the user will not have to use any flag.
For local images, user will have to set `containerRegistry` to an empty string (which of course will be documented).

Reply:
Ok, by the way the images are already in the ghcr : https://github.com/orgs/clusterlink-net/packages